### PR TITLE
Add new Go versions

### PR DESCRIPTION
**Pull request by paultag**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 language: go
 go_import_path: pault.ag/go/debian
 go:
+  - 1.9
+  - 1.8
+  - 1.7
   - 1.6
   - 1.5
```
Comment (tianon) [inline .travis.yml]: It's relatively new, but I'd recommend adding `.x` to all these so you get the latest point release. :+1:

Comment (tianon) [reply .travis.yml]: See https://docs.travis-ci.com/user/languages/go/#Specifying-a-Go-version-to-use :+1:

Comment (paultag) [reply .travis.yml]: Nice call! Thanks! Added a `.x` to them all!

**Comment by tianon** [comment]
What a great phone I have, making such colorful emoji out of :us:.

**Pull request merged**